workflow-trace-viewer/src/jvmMain/kotlin/com/squareup/workflow1/traceviewer/App.kt

```diff
@@ -1,9 +1,44 @@
 package com.squareup.workflow1.traceviewer
 
+import androidx.compose.foundation.layout.Box
 import androidx.compose.material.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import io.github.vinceglb.filekit.PlatformFile
+import io.github.vinceglb.filekit.readString
 
+/**
+ * Main composable that provides the different layers of UI.
+ */
 @Composable
-fun App() {
-  Text("Hello world!")
+public fun App(
+  modifier: Modifier = Modifier
+) {
+  Box {
+    val selectedFile = remember { mutableStateOf<PlatformFile?>(null) }
```

Comment:
Prefer using Kotlin delegation here.

```kotlin
var selectedFile by remember { mutableStateOf<PlatformFile?>(null) }
```

That'll let you do simple reassignments without having to do the whole `.value` stuff to access or change the value. Android Studio will try to prompt you to import the `getValue` and `setValue` extension methods that make that possible, but if it doesn't, you may need to manually import them. 

An example of this syntax is here: https://github.com/squareup/android-emulator-runner/blob/92d600d5e9cb215e3509ee215881e55a42fb6fac/desktop-app/src/jvmMain/kotlin/com/wardellbagby/aer/ui/App.kt#L192

The imports you _may_ need to add manually are:

```kotlin
import androidx.compose.runtime.getValue
import androidx.compose.runtime.setValue 
```